fe/src/main/java/org/apache/doris/analysis/Expr.java

```diff
@@ -123,6 +123,24 @@ public boolean apply(Expr arg) {
                 }
             };
     */
+    // Returns true if an Expr is a builtin aggregate function.
+    public final static com.google.common.base.Predicate<Expr> IS_BUILTIN_AGG_FN =
```

Comment:
The name is a bit inaccurate, bitmap_union and hll_union are builtin aggregate function too.
So could you choose another name?